vscode/src/test/suite/ruby/chruby.test.ts

```diff
@@ -318,4 +318,36 @@ suite("Chruby", () => {
       return chruby.activate();
     });
   });
+
+  test("Uses closest Ruby if the version specified in .ruby-version is not installed (patch difference)", async () => {
+    fs.writeFileSync(path.join(workspacePath, ".ruby-version"), "ruby '3.3.3'");
+
+    const chruby = new Chruby(workspaceFolder, outputChannel, async () => {});
+    chruby.rubyInstallationUris = [
+      vscode.Uri.file(path.join(rootPath, "opt", "rubies")),
+    ];
+
+    const { env, version, yjit } = await chruby.activate();
+
+    assert.match(env.GEM_PATH!, new RegExp(`ruby/${VERSION_REGEX}`));
+    assert.match(env.GEM_PATH!, new RegExp(`lib/ruby/gems/${VERSION_REGEX}`));
+    assert.strictEqual(version, RUBY_VERSION);
+    assert.notStrictEqual(yjit, undefined);
```

Comment:
Is `yjit` relevant here?

Reply:
Yes, we want to check that YJIT was set to something.